apolloschurchapp/appcenter-post-clone.sh

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
We should probably leave these in for now. I don't want them to be accidentally deleted from our client repos when they update and cause confusion and force a premature migration to Github Actions.

Reply:
fair. I could a separate PR to remove these pointed to next